README.md

```diff
@@ -1,21 +1,59 @@
-# 18F Open Source Policy
+# datagov-ssb
 
-This repository contains the official [Open Source Policy](policy.md) of [18F](https://18f.gsa.gov/) (a digital delivery team within the [General Services Administration](http://gsa.gov)).
+The Supplementary Service Broker (SSB) fills gaps in cloud.gov's brokered
+services.
 
-**[Read 18F's open source policy.](policy.md)**
+Services are defined in a
+[brokerpaks](https://github.com/pivotal/cloud-service-broker/blob/master/docs/brokerpak-intro.md),
+bundles of Terraform and YAML that specifies the service should be
+advertised, provisioned, bound, unbound, and unprovisioned.
 
-### 18F Team Guidance
+# Dependencies
```

Comment:
Minor, but there should be only one h1 level header on the page. This and following should probably just be h2's
```suggestion
## Dependencies
```

Reply:
Will address in a big README revamp I'm working on anyway.